docs/validators/setup/mempool.md

```diff
@@ -0,0 +1,69 @@
+<!--
+order: 5
+-->
+
+# Mempool
+
+Learn about the available mempool options in Tendermint.{synopsis}
+
+## FIFO Mempool
+
+The mempool holds uncommitted transactions, which are not yet included in a block.
+The default mempool implementation for Tendermint blockchains follows a first-in-first-out (FIFO) principle,
+which means the ordering of transactions depends solely on the order in which they arrive at the node.
+The first transaction to be received will be the first transaction to be processed.
+This is true for gossiping the received transactions to the rest of the peers as well as including them in a block.
+
+## Prioritized Mempool
+
+Starting with [Tendermint v0.35](https://github.com/tendermint/tendermint/blob/v0.35.0/CHANGELOG.md)
+(has also been backported to [v0.34.20](https://github.com/tendermint/tendermint/blob/17c94bb0dcb354c57f49cdcd1e62f4742752c803/UPGRADING.md?plain=1#L54))
+it is possible to use a prioritized mempool implementation.
+This allows validators to choose transactions based on the associated fees or other incentive mechanisms.
+It is achieved by passing a `priority` field with each [`CheckTx` response](https://github.com/tendermint/tendermint/blob/17c94bb0dcb354c57f49cdcd1e62f4742752c803/proto/tendermint/abci/types.proto#L234),
+which is run on any transaction trying to enter the mempool.
+
+Evmos supports [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559#simple-summary) EVM transactions through its
+<!-- markdown-link-check-disable-next-line -->
+[feemarket](../../modules/feemarket/01_concepts.md) module.
+This transaction type uses a base fee and a selectable priority tip that add up to the total transaction fees.
+The prioritized mempool presents an option to automatically make use of this mechanism regarding block generation.
```

Comment:
This looks a bit off, check the "Rich diff" setting to see how it looks

Reply:
Renders fine when building locally:
<img width="810" alt="image" src="https://user-images.githubusercontent.com/42640438/205293750-ed7ea36f-5b28-4aed-b75f-6f7737a0fc85.png">